cpp/CMakeLists.txt

```diff
@@ -47,6 +47,10 @@ if(POLICY CMP0074)
   cmake_policy(SET CMP0074 NEW)
 endif()
 
+if(POLICY CMP0114)
+  cmake_policy(SET CMP0114 NEW)
```

Comment:
Would you mind adding a comment explaining why this is necessary? So that future readers are not confused by this.

Reply:
Added a comment, I don't know the exact details, just that CMake complains vocally with a lot of yellow text if it's not set (or set to OLD).